### ISSUE TITLE
ColorPicker on a touchscreen

### ISSUE THREAD

**Issue by mfauvain** [flameshot-org/flameshot#1798]
Hi, I've started to use flameshot and it's amazing. However on a touchscreen (with no keyboard attached), launching colorpicker with spacebar can't work.
Any chance to add the color picker to the available tools?

**Comment by mmahmoudian** [comment]
I see your point and I will label it as Enhancement (especially considering that the functionality is there and just a button in among the tools should be created for it), but I wonder why don't you  do the following:
![image](https://user-images.githubusercontent.com/390889/127974551-967c134a-aaa9-4543-abfb-1324ad7cfff4.png)

**Comment by mfauvain** [comment]
Agree, can be done this way, but I like the 'gnome-pie' lookalike better, it is easier to quickly pick a color.
I would even add an option to have similar tools (line, shapes, etc..) in a similar gnome-pie like to reduce the number of tools around the cut window.

**Comment by borgmanJeremy** [comment]
@mmahmoudian seeing as we just merged the rework for the color picker and we kept it in the sidebar, I think it is unlikely we will add it to the tool list. Should we close this?

**Comment by mmahmoudian** [comment]
@borgmanJeremy I kinda also see some merit into this and would be nice if we can have a bubble in the right-click color wheel to call that color grabber that @veracioux just developed. Basically I'm not the one who should decide as you and @veracioux  know the code based and if it is easy to integrate. Otherwise I believe we should close this feature request although we have not fulfilled it.

**Comment by veracioux** [comment]
@mmahmoudian @borgmanJeremy
I designed the interface for the color grabber to be embeddable anywhere (anticipating a PR for #328 in the future). Or, at least it should be with small tweaks. And adding a button to the color picker pie shouldn't be too hard if I'm not mistaken.

I'm not a touchscreen user so I can't particularly endorse this feature. But I do think the pie would be the most logical place for the color grabber. So I vote to leave this open. I'll see how much trouble it would be to add this feature and I'll get back to you. (though you may have to wait a bit depending on my schedule)

@mfauvain 
> I would even add an option to have similar tools (line, shapes, etc..) in a similar gnome-pie like to reduce the number of tools around the cut window.

I believe this would cause more developer trouble than user merit.

**Comment by mfauvain** [comment]
I agree.. It's a great tool as it is and using it quite extensively now. Thx
